apisix/plugins/google-cloud-logging.lua

```diff
@@ -62,7 +63,7 @@ local schema = {
                     default = "https://logging.googleapis.com/v2/entries:write"
                 },
             },
-            required = { "private_key", "project_id", "token_uri" }
+            required = { "client_email", "private_key", "project_id", "token_uri" }
```

Comment:
Could you show me the official doc that the google cloud logging need the `client_email`?

Reply:
From the JWT claim set
https://developers.google.com/identity/protocols/oauth2/service-account?hl=en#authorizingrequests
![image](https://github.com/apache/apisix/assets/9668312/65d45efa-15bc-4150-b0d6-811fbfc4e25f)